.setup/INSTALL_SUBMITTY_HELPER.sh

```diff
@@ -477,13 +477,29 @@ g++ ${SUBMITTY_INSTALL_DIR}/src/grading/system_call_check.cpp -o ${SUBMITTY_INST
 # build the helper program for calculating early submission incentive extensions
 g++ ${SUBMITTY_INSTALL_DIR}/bin/calculate_extensions.cpp -lboost_system -lboost_filesystem -std=c++11 -Wall -g -o ${SUBMITTY_INSTALL_DIR}/bin/calculate_extensions.out
 
+GRADINGCODE=${SUBMITTY_INSTALL_DIR}/src/grading
+JSONCODE=${SUBMITTY_INSTALL_DIR}/vendor/include
+
+# Create the complete/build config using main_configure
+g++ ${GRADINGCODE}/main_configure.cpp ${GRADINGCODE}/load_config_json.cpp ${GRADINGCODE}/execute.cpp \
```

Comment:
I'm not at all knowledgeable about this part of the codebase so I don't know how much performance matters, but is there any reason to not add some sort of optimization flag here?

Reply:
Good point. I did some quick performance testing to see if it would be worth it. Before/after is comparing adding the -O3 flag so before is without any optimizations.

Compilation time:
Before: 26 seconds
After: 42.8 seconds (and it threw a couple of warnings)

Running time (of only configure.out):
Before: 0.006 seconds
After: 0.007 seconds

Note that these aren't averaged so the running time is probably incorrect but the point I'm trying to make is that the values are so close it's probably not worth the extra compilation time unless we were building thousands of gradeables.